app/pages/analyst/application/[applicationId]/assessments.tsx

```diff
@@ -20,15 +32,55 @@ const getAssessmentsQuery = graphql`
 const Assessments = ({
   preloadedQuery,
 }: RelayProps<Record<string, unknown>, assessmentsQuery>) => {
-  const { applicationByRowId, session } = usePreloadedQuery(
-    getAssessmentsQuery,
-    preloadedQuery
+  const query = usePreloadedQuery(getAssessmentsQuery, preloadedQuery);
+
+  const { applicationByRowId, session } = query;
+  const [formData, setFormData] = useState(
+    applicationByRowId.assessmentForm?.jsonData
   );
+  const [createAssessment, isCreating] = useCreateScreeningAssessmentMutation();
+
+  const handleSubmit = async (newFormData: any) => {
+    createAssessment({
+      variables: {
+        input: {
+          _applicationId: applicationByRowId.rowId,
+          _jsonData: newFormData,
+          schemaSlug: 'screeningAssessmentSchema',
+        },
+      },
+      onCompleted: () => {},
+    });
+  };
 
   return (
     <Layout session={session} title="Connecting Communities BC">
       <AnalystLayout application={applicationByRowId}>
         <h2>Assessments placeholder</h2>
+        <div>
+          {/* Tabs section here */}
+          Screening
+        </div>
+        <FormBase
+          schema={screening}
+          uiSchema={screeningUiSchema}
+          noValidate
+          onChange={(e) => {
+            setFormData(e.formData);
+          }}
+          formData={formData}
+          formContext={{ query }}
+          tagName="div"
```

Comment:
I have good but annoying news, all the source of our troubles was this `tagName="div"` which changes the `Form` to a `div`. The reason this was brought over for the analyst application page is it is taken from the review `FormBase` which we don't want to submit by default.